terraform/terraform_spec.rb

```diff
@@ -23,6 +23,14 @@
     expect(file("/etc/alpine-release")).to be_file
   end
 
+  it "installs the right version of Alpine" do
+    expect(os_version).to include("Alpine Linux 3.3")
+  end
```

Comment:
If we stop checking the Alpine version should we also remove the Terraform version check on line 55?